pkg/networkplugin-syncer/handlers/ovn/nbctl/nbctl.go

```diff
@@ -103,9 +103,9 @@ func (n *NbCtl) GetGatewayChassis(lrp, chassis string) (string, error) {
 	return output, err
 }
 
-func (n *NbCtl) LrSetLoadBalancers(lrId string, lbIDs []string) error {
+func (n *NbCtl) LrSetLoadBalancers(lrID string, lbIDs []string) error {
```

Comment:
Although the linter doesn’t flag this, `Lr` should be `LR` throughout since it’s an acronym.

Reply:
Ignoring for now.